### DISCUSSION TITLE
Understanding the Hackathon

### DISCUSSION THREAD

**Discussion by sinairusinek** [bigscience-workshop/lam#63] [General]
Hi all!
I'm thinking of two datasets that might fit the idea of the Lam bigscience initiative, but can't really understand the idea of the Hackathon in it. Is it simply a time frame to get us all to collect these datasets and document them? Or is there some hacking -in the sense of prototyping, processing and analysis -  that is going to take place, and if so - where can I find more details about it?

**Comment by davanstrien** [comment]
Hi @sinairusinek!

The 'hackathon' in the name is mainly to indicate that we have some limited time frame for this activity.

However we would love to see experiments, analysis etc with any of the datasets uploaded as part of this sprint. If you have any ideas about things you want to work on in this area feel free to mention them below. 

A lot of the datasets are likely to be fairly underexploited compared to more well known machine learning datasets so I suspect there are still many things to discover about the best way to work with them.

Hope that helps and feel free to ask if you have more questions or ideas 🙂

**Reply by sinairusinek** [reply]
Thanks! The dataset we have in mind is a scanned historic archive that we are working on - currently focusing on Image classification, but planning also documents reconstruction (grouping images to documents) and image segmentation, towards OCR.  It would be good to have a few tips on how an image dataset should be built for HF, similar to the explanation on text datasets! (e.g. should it include only annotated images?)
